### PR TITLE
Move ingress to v1

### PR DESCRIPTION
**Pull request by QuentinBisson**
This needs to be tested

## Checklist

I have:

- [x] Described why this change is being introduced
- [x] Separated out refactoring/reformatting in a dedicated PR
- [x] Updated changelog in `CHANGELOG.md`

### CODE REVIEW

**Review by TheoBrigitte — APPROVED** [review]
LGTM when tested

File: service/controller/resource/verticalpodautoscaler/resource.go
Comment (mig4) [inline service/controller/resource/verticalpodautoscaler/resource.go:105]: Is this related to ingress somehow?

**Review by mig4 — APPROVED** [review]

**Comment by QuentinBisson** [comment]
@mig4 I am thinking this will be blocked until nginx is on the latest versions everywhere though. Is there a way to upgrade to the latest version?

**Comment by mig4** [comment]
> @mig4 I am thinking this will be blocked until nginx is on the latest versions everywhere though. Is there a way to upgrade to the latest version?

😱 really? damn, I think that means another upgrade of _g8s-terraform_ and also mayu this time 😭

**Comment by QuentinBisson** [comment]
Yes, because nginx ingress controller supported v1 in 0.40 https://github.com/kubernetes/ingress-nginx/releases/tag/controller-v0.40.0

**Comment by QuentinBisson** [comment]
Superseded by https://github.com/giantswarm/prometheus-meta-operator/pull/826

**Pull request closed without merging**
